var/spack/repos/builtin/packages/ossp-uuid/package.py

```diff
@@ -22,7 +24,8 @@ class OsspUuid(AutotoolsPackage):
     depends_on("c", type="build")  # generated
     depends_on("cxx", type="build")  # generated
 
-    provides("uuid")
+    if sys.platform not in ["darwin", "windows"]:
```

Comment:
`sys.platform` reports `"win32"` on Windows, not "windows"

Reply:
oops sorry about that. resolved